### PR TITLE
Add ActivityPub in the protocol list in dfrn_request

### LINKED ISSUE #8234: dfrn_request.php has really outdated information
image

GNU Social should probably say OStatus and there should be a new "network" that says something like 
ActivityPub (Pleroma, Mastodon, PixelFed, MissKey, etc.)

thread about it on the fediverse posted [here](https://social.wake.st/@liaizon/103602686848917793)

### PR DESCRIPTION
**Pull request by MrPetovan**
Closes #8234

### CODE REVIEW

**Comment by annando** [comment]
We cannot guarantee that the remote subscription stuff will work with all AP implementations. So I suggest changing the text completely, like: "Enter your handle or profile URL here. If this isn't supported by your system (for example it doesn't work with Diaspora), you have to subscribe user@domain.tld directly on your system"

**Comment by MrPetovan** [comment]
Speaking of remote subscription working with Mastodon, could you please have a look at this? https://github.com/friendica/friendica/issues/8233

Got it for the alternative text.

**Comment by MrPetovan** [comment]
Currently:

> Please enter your 'Identity address' from one of the following supported communications networks:
> 
> - [Friendica](http://friendi.ca/)
> - [ActivityPub (Mastodon, Pleroma, Pixelfed, MissKey, PeerTube...)](https://activitypub.rocks/)
> - [diaspora* (Socialhome, Hubzilla)](https://diasporafoundation.org/) - please do not use this form. Instead, enter user@domain.tld into your diaspora* search bar.
> - [OStatus (GNU Social)](https://gnu.io/social/)
> If you are not yet a member of the free social web, [follow this link to find a public Friendica node and join us today](https://dir.friendica.social/servers).
> 
> Examples: jojo@demo.friendi.ca, http://demo.friendi.ca/profile/jojo, user@gnusocial.de

New version:

> Enter your handle or profile URL here. If this isn't supported by your system (for example it doesn't work with Diaspora), you have to subscribe to **user@domain.tld** directly on your system

Or did you want to keep anything from the old copy, like the list of supported protocols or the link to the server directory?

**Comment by annando** [comment]
I'm fine with the changes. Next step would be to handle Friendica connects like all the other connects as well. Currently we do have a special treatment here. But this would be a task for a different PR.

**Comment by MrPetovan** [comment]
New design:
friendica_dfrn_request_redesign

File: mod/follow.php [hunk 1]
```diff
@@ -163,29 +163,22 @@ function follow_content(App $a)
...
-		'$page_desc'     => '',
-		'$friendica'     => '',
-		'$statusnet'     => '',
-		'$diaspora'      => '',
-		'$diasnote'      => '',
 		'$your_address'  => DI::l10n()->t('Your Identity Address:'),
-		'$invite_desc'   => '',
-		'$emailnet'      => '',
+		'$url_label'     => DI::l10n()->t('Profile URL'),
+		'$keywords_label'=> DI::l10n()->t('Tags:'),
```
Comment (annando) [inline mod/follow.php]: What are the tags for?

Comment (MrPetovan) [reply mod/follow.php]: They show the public tags of the profile you're about to follow.

**Pull request merged**